File: src/test/fixtures/typescript/opaqueBoolean.ts
// flagged — opaque boolean
function flaggedPositionalBoolean() {
    configure(true);
}

// flagged — opaque boolean
function flaggedPositionalBooleanAmongOthers() {
    process(1, false);
}

// clean — not flagged by opaque boolean
function suppressedObjectLiteralField() {
    configure({ retries: true });
}

// clean — not flagged by opaque boolean
function suppressedNonCallUsage() {
    const ok = true;
    return ok;
}

File: src/test/fixtures/kotlin/OpaqueBoolean.kt
// flagged — opaque boolean
fun flaggedPositionalBoolean() {
    configure(true)
}

// flagged — opaque boolean
fun flaggedPositionalBooleanAmongOthers() {
    process(1, false)
}

// clean — not flagged by opaque boolean
fun suppressedNamedArgument() {
    configure(retries = true)
}

// clean — not flagged by opaque boolean
fun suppressedNonCallUsage(): Boolean {
    return true
}
